lnwallet/interface.go

```diff
@@ -636,6 +636,16 @@ func InternalKeyForAddr(wallet WalletController, netParams *chaincfg.Params,
 
 	walletAddr, err := wallet.AddressInfo(addr)
 	if err != nil {
+		// If the error is that the address can't be found, it is not
+		// an error, as callers can use the .Option() method to get an
```

Comment:
Not sure I follow here, could you elaborate a bit? If there's no internal key associated shouldn't we return an error?

Reply:
I changed the comment:

```go
        walletAddr, err := wallet.AddressInfo(addr)
        if err != nil {
                // If the error is that the address can't be found, it is not
                // an error. This happens when any channel which is not a custom
                // taproot channel is cooperatively closed to an external P2TR
                // address. In this case there is no internal key associated
                // with the address. Callers can use the .Option() method to get
                // an option value.
```